src/main/java/io/strimzi/kafka/bridge/mqtt/config/KafkaProducerConfig.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.bridge.mqtt.config;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * Represents configurations related to Kafka
+ * @see AbstractConfig
+ */
+public class KafkaProducerConfig extends AbstractConfig {
+
+    // Prefix for all the specific configuration parameters for Kafka producer in the properties file
+    public static final String KAFKA_PRODUCER_CONFIG_PREFIX = KafkaConfig.KAFKA_CONFIG_PREFIX + "producer.";
+
+    // Acknowledgement level for Kafka producer
+    public static final String ACKS_LEVEL = KAFKA_PRODUCER_CONFIG_PREFIX + "acks";
```

Comment:
we don't want a specific constant for ALL the possible Kafka producer config parameters. If you open the Kafka doc they are a lot but in any case it doesn't make sense. Everything coming after `kafka.producer.` is put inside the map in the base class and then passed to the Kafka producer constructor. You can just follow the Strimzi HTTP bridge pattern here, without adding this.

Reply:
@ppatierno in HTTP bridge some Kafka related parameter constants are from `org.apache.kafka.clients` package and I thought maybe I should not add any dependency related to Kafka yet. But if you are okay, I can go ahead.

Reply:
Why do you need Kafka clients dependency right now? You are not going to create a Kafka producer for now.

Reply:
If you take a look at how the HTTP bridge works, all bridge parameters for the Kafka producer are prefixed with `kafka.producer.` and the last part has the name of a Kafka producer parameter.
For example, if you set `kafka.producer.acks` in the bridge configuration, the `acks` part is the name of a parameter for the underneath Kafka producer directly (see the Kafka documentation for that). You don't need a constant for it.

Reply:
(here)[https://github.com/strimzi/strimzi-kafka-bridge/blob/eab72b43d3b63742a2ffa3da2891235ab310564c/src/test/java/io/strimzi/kafka/bridge/ConfigRetrieverTest.java#L44] the ACK is coming from `import org.apache.kafka.clients.producer.ProducerConfig`

Reply:
but I have removed the constant! and just kept one prefix